compiler/lib/reserved.ml

```diff
@@ -140,7 +140,24 @@ let provided =
     ; "isNaN"
     ; "parseFloat"
     ; "parseInt"
-    ; "module"
-    ; "require"
+    ; "module" (* only available in node *)
+    ; "require" (* only available in node *)
     ; "Symbol"
+    ; "ArrayBuffer"
+    ; "Float32Array"
+    ; "Float64Array"
+    ; "Int16Array"
+    ; "Int32Array"
+    ; "Int8Array"
+    ; "TextDecoder"
+    ; "Uint16Array"
+    ; "Uint32Array"
+    ; "Uint8Array"
+    ; "atob"
```

Comment:
If we include `atob`, we should also include `btoa`

Reply:
Good idea! Added!